orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueue.kt

```diff
@@ -104,8 +107,13 @@ class RedisQueue(
   override fun push(message: Message, delay: TemporalAmount) {
     pool.resource.use { redis ->
       val messageHash = message.hash()
-      if (redis.sismember(hashesKey, messageHash)) {
-        log.warn("Ignoring message as an identical one is already on the queue: $message")
```

Comment:
I'm also curious what code is pushing the duplicate message ... dumping a stack trace seems heavyweight tho.